packages/main/src/SegmentedButton.js

```diff
@@ -26,11 +26,11 @@ const metadata = {
 	slots: /** @lends sap.ui.webcomponents.main.SegmentedButton.prototype */ {
 
 		/**
-		 * Defines the buttons of component.
+		 * Defines the buttons of <code>ui5-segmentedbutton</code>.
 		 * <br><br>
 		 * <b>Note:</b> Multiple buttons are allowed.
 		 * <br><br>
-		 * <b>Note:</b> Use the <code>ui5-toggle-button</code> for the intended design.
+		 * <b>Note:</b> Use the <code>ui5-segmented-button-item</code> for the intended design.
```

Comment:
We have ui5-segmentedbutton as  wrapping control, but ui5-segmented-button-item as inner, don't we need to have ui5-segmentedbutton-item? It will be more consistent.